src/components/editor/EditorArticleController.jsx

```diff
@@ -56,7 +57,7 @@ export default class EditorArticleController extends FalcorController {
   }
 
   static getFalcorPathSets(params) {
-    return [['articlesBySlug', params.slug, ['title', 'issue', 'category', 'description']], ['articlesBySlug', params.slug, 'authors', {to: 2}, ['slug', 'name']]];
+    return [['articlesBySlug', params.slug, ['title', 'issue', 'category', 'description']], ['articlesBySlug', params.slug, 'authors', {to: 2}, ['slug', 'name']], ['latestIssue']];
```

Comment:
We can do it reverse chronological so `issues[0]` always returns the newest issue.